### PR TITLE
Completed css-grid-classes -note i need to slow down

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: center;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:10]: This is not a flex container, so this has no effect, and should be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Owen! Good work so far, one minor fix:

**Comment by owen-adams2021** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nicely done!! 👍

**Pull request merged**